tests/transport/test_requests.py

```diff
@@ -84,44 +86,44 @@ def test_constructor(self):
         assert authed_session.credentials == mock.sentinel.credentials
 
     def test_request_no_refresh(self):
-        mock_credentials = mock.Mock(wraps=MockCredentials())
-        mock_response = make_response()
-        mock_adapter = MockAdapter([mock_response])
+        credentials = mock.Mock(wraps=CredentialsStub())
```

Comment:
Because in this case I want the partial functionality of the stub as well as the ability to assert that calls were made.